jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml

```diff
@@ -7,7 +7,7 @@
             days-to-keep: 7
     provider-env: |
         export KUBERNETES_PROVIDER="gce"
-        export E2E_MIN_STARTUP_PODS="1"
+        export E2E_MIN_STARTUP_PODS="8"
```

Comment:
I can remove this change to the kubemark parameters.
Original change was in global.yaml but after rebase, I tried to add it to all provider environments.